docs/ref/contrib/postgres/forms.txt

```diff
@@ -199,3 +199,28 @@ DateRangeField
     Based on :class:`~django.forms.DateField` and translates its input into
     :class:`~psycopg2:psycopg2.extras.DateRange`. Default for
     :class:`~django.contrib.postgres.fields.DateRangeField`.
+
+.. currentmodule:: django.contrib.postgres.widgets
+
+Widgets
+------------
+
+RangeWidget
+~~~~~~~~~~~~~~
+
+.. class:: RangeWidget(base_widget, attrs=None)
+
+    Widget used by all of the Range Fields as mentioned above.
```

Comment:
"range fields" with no caps
chop "as mentioned above"
Add "All of these fields use RangeWidget." above in the description of "Ranges Fields" after "not greater than the upper bound."